assets/cubyz/biomes/ferrock_mountains.zig.zon

```diff
@@ -6,13 +6,23 @@
 	},
 	.minHeight = 80,
 	.maxHeight = 256,
-	
+	.minRadius = 100,
+	.maxRadius = 160,
```

Comment:
Making the radius so small, makes the transitions really harsh:
![Screenshot at 2025-01-08 23-20-56](https://github.com/user-attachments/assets/34ebf24c-ab0f-4797-8c10-c4d646630752)
Furthermore it makes the biome selection more random when multiple biomes share the same climate parameters.

Reply:
It still doesn't seem to be fixed. I still see many places where the "mountain" is just a flat wall:
![Screenshot at 2025-01-09 17-54-22](https://github.com/user-attachments/assets/b3534dac-45bc-414b-9877-701fee7afcd7)

Reply:
The interpolation depends on the biome size. If the biomes are too small, and a radius of 100 is too small, then it can only interpolate on a small range.
If I change the biome size of the desert and the ferrock mountains to 500 then the transition is much smoother:
![Screenshot at 2025-01-09 22-12-07](https://github.com/user-attachments/assets/f47e53d2-c56c-43f8-be4e-91fc40f2f742)